src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/missingdeprecated/InputMissingDeprecatedSkipNoJavadoc.java

```diff
@@ -1,27 +0,0 @@
-package com.puppycrawl.tools.checkstyle.checks.annotation.missingdeprecated;
-
-public class InputMissingDeprecatedSkipNoJavadoc {
```

Comment:
Please explain why you remove input.
Is it a duplicate ?

Reply:
As input is titled, it was only for the property that we are now removing, `skipNoJavadoc`.
Do you wish to keep the test even though it will have no meaning now on?